README.md

```diff
@@ -1 +1,7 @@
-# lua-func
\ No newline at end of file
+# lua-func
+
+The lua interpreter split in a way such that each function is in it's own file.
+
+However not all of the tests pass.
+
+One of the popen tests in files.lua:747 is comented out, and the entire main.lua file is ignored. This amounts to roughly 15% of the tests failing.
```

Comment:
Line needs wrapping.

Reply:
Do we need the text wrapping for markdown files, which will be mostly viewed in the browser which provides better line wrapping?

Reply:
Indeed we do need to wrap -- long lines work very badly with `diff`.